src/fix.js

```diff
@@ -19,11 +19,10 @@ function fix(n, encoding, isWrapped) {
         fixed[i] = fix(n[i], encoding);
       }
     } else {
-      if (n.constructor === BigNumber) {
-        fixed = n.mul(FXP_ONE).round();
-      } else {
-        fixed = bignum(n).mul(FXP_ONE).round();
+      if (n.constructor !== BigNumber) {
```

Comment:
Should this be `BigNumber.isBigNumber` now?

Reply:
Oh that's true! Replacing